plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/model/LibraryFactory.java

```diff
@@ -81,9 +81,9 @@ static Library create(IConfigurationElement configurationElement) throws Library
           library.setExport(Boolean.parseBoolean(exportString));
         }
         String dependencies = configurationElement.getAttribute("dependencies"); //$NON-NLS-1$
-        if ("include".equals(dependencies)) { //$NON-NLS-1$
+        if (!"include".equals(dependencies)) { //$NON-NLS-1$
           // Load the dependencies from Maven Central later
```

Comment:
The comment doesn't seem right, if we look at the code of `setResovled()`. The method makes `library` "final" so that it won't attempt to resolve transitive dependencies.

Reply:
vestigial; removed